ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -392,3 +392,65 @@ def set_upgrade_images(self):
             with NamedTemporaryFile() as cs_yaml:
                 dump_data_to_temp_yaml(cs_data, cs_yaml.name)
                 ocs_catalog.apply(cs_yaml.name)
+
+
+def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
+    """
+    Run upgrade procedure of OCS cluster
+
+    Args:
+        operation: (function): Function to run
+        operation_args: (iterable): Function's arguments
+        operation_kwargs: (map): Function's keyword arguments
+
+    """
+
+    ceph_cluster = CephCluster()
+    upgrade_ocs = OCSUpgrade(
+        namespace=config.ENV_DATA['cluster_namespace'],
+        version_before_upgrade=config.ENV_DATA.get("ocs_version"),
+        ocs_registry_image=config.UPGRADE.get('upgrade_ocs_registry_image'),
+        upgrade_in_current_source=config.UPGRADE.get('upgrade_in_current_source', False)
+    )
+    upgrade_version = upgrade_ocs.get_upgrade_version()
+    assert upgrade_ocs.get_parsed_versions()[1] >= upgrade_ocs.get_parsed_versions()[0], (
+        f"Version you would like to upgrade to: {upgrade_version} "
+        f"is not higher or equal to the version you currently running: "
+        f"{upgrade_ocs.version_before_upgrade}"
+    )
+    csv_name_pre_upgrade = upgrade_ocs.get_csv_name_pre_upgrade()
+    pre_upgrade_images = upgrade_ocs.get_pre_upgrade_image(csv_name_pre_upgrade)
+    upgrade_ocs.load_version_config_file(upgrade_version)
+    with CephHealthMonitor(ceph_cluster):
+        channel = upgrade_ocs.set_upgrade_channel()
+        upgrade_ocs.set_upgrade_images()
+        upgrade_ocs.update_subscription(channel)
+        if operation:
+            log.info(f"Calling test function: {operation}")
+            _ = _ = operation(*operation_args, **operation_kwargs)
```

Comment:
```suggestion
            _ = operation(*operation_args, **operation_kwargs)
```

I think that one is extra here, right?

Reply:
typo, yes. fixed